modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/evaluation.es.js

```diff
@@ -192,7 +192,7 @@ const doEvaluate = debounce((fieldName, evaluatorContext, callback) => {
 			callback(null, mergedPages);
 		})
 		.catch((error) => callback(error));
-}, 200);
+}, 350);
```

Comment:
@larissacribeiro @igor-franca, sorry, in my last review comment I forgot to point to the new value proposed by @rebsilva.